src/scripts/main.js

```diff
@@ -1,3 +1,76 @@
 'use strict';
 
-// write code here
+const refs = {
+  buttons: {
+    appendRow: document.querySelector('.append-row'),
+    removeRow: document.querySelector('.remove-row'),
+    appendCln: document.querySelector('.append-column'),
+    removeCln: document.querySelector('.remove-column'),
+  },
+  table: document.querySelector('.field'),
+  row: document.querySelector('.field tr'),
```

Comment:
The `refs.row` variable is defined but never used in the code. Consider removing it if it's unnecessary, or utilize it if it was intended for a specific purpose.